### DISCUSSION TITLE
Add api for accessing recipes (CraftingManager?)

### DISCUSSION THREAD

**Discussion by STPv22** [eaglerforge/EaglerForge#46] [Ideas]
Ode de linux wanted this in a mod they made so they dont have to grab a json file of the recipes.

**Comment by STPv22** [comment]
Looks like its FurnaceRecipes and a couple other files that have them as well

**Comment by Block2222** [comment]
upvote, i see how this can become really useful

**Reply by STPv22** [reply]
welp, now with EFI, we don't need this

**Comment by STPv22** [comment]
closing don't listen to me
